Makefile

```diff
@@ -51,6 +51,10 @@ ARCH     ?= rv32imc
 # Path relative from the location of sw/Makefile from which to fetch source files. The directory of that file is the default value.
 SOURCE 	 ?= "."
 
+# Number of memory banks. Set to 3 by default to allow the testing of ram retention examples
+MEMORY_BANKS 	?= 3
```

Comment:
this I cannot accept it :) must be 2 otherwise we break compatibility with existing systems

Reply:
You mean any project should now modify their external Makefile to force mcu-gen to include `MEMORY_BANKS=2`. 
I can agree thats far from nice. 

Options I see are:  
1. We use 3 mb by default (requires project depending on the 2 mb to make a change on their external Makefile) 
2. We find a way of making the `set_retentive` apps work with 2 mb (no idea if this is even feasible) 
3. We make all tests with `MEMORY_BANKS=3` (but if someone trying an app as-is -with 2 mb-, it might not work)
4. We test only these apps with `MEMORY_BANKS=3` (it would slow down testing and pose a slight challenge if we try to automate tests)

Opinions?   @davideschiavone @simone-machetti

Reply:
This change was overturned